doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -1023,6 +1023,14 @@ Requesting information
    fixpoint and cofixpoint is violated at some time of the construction
    of the proof without having to wait the completion of the proof.
 
+.. cmd:: Validate Proof
+
+   This command checks that the current partial proof is well-typed.
```

Comment:
```suggestion
   Checks that the current partial proof is well-typed.
```

It will do the right thing for nested proofs, right?  ie only checks the nested proof.